.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ["bug"]
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
```

Comment:
```suggestion

**Map save file**
Please attach save file where the issue can be reproduced

**Game logs**
Please attach game logs: `VCMI_client.txt`, `VCMI_server.txt` etc.

```

Reply:
I get the idea... but from my experience this may be counter-productive.

I think we should prefer repro steps in following order:
- for generic bug - simply describe steps to reproduce
- for bugs that appear on specific map - said map and steps to reproduce
- for bugs that only appear in the same - said save and repro steps
- for non-reproducible crashes - crash dump + logs

Asking for saves may lead to non-reproducible reports simply because of save compatibility issues. Even if report was about something that could have easily happen on any map

Oh, totally agree on logs though.
```suggestion

**Game logs**
Please attach game logs: `VCMI_client.txt`, `VCMI_server.txt` etc.

```

Reply:
ok, let's have logs at least